content/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+**What**: Explain what you changed and provide a list of changed page names.
+
+**Why:** Explain why this change is necessary and how it benefits users.
+
+**Screenshots:** Optional. Show additions to the sidebar or new formatting.
+
+----------
+
+### Merge Checklist
+_If items do not apply to your changes, add (N/A) and mark them as complete._
+
+**Pull Request**
+- [ ] One or more labels describe the type of change (e.g. clarification) and associated product (e.g. tfc).
```

Comment:
Can we make all the checklist start with verbs? I think it's more clear/explicit for the reader to understand what's expected of them. This would also mirror how we do it in the [tutorials](https://github.com/hashicorp/tutorials/blob/main/.github/pull_request_template.md) repo

```suggestion
- [ ] Added at least one label to describe the type of change (e.g. clarification) and associated product (e.g. tfc).
```

Reply:
I thought about this, but I feel like that would result in a lot of similar/empty verbs at the start of each bullet. And I kind of want to lead with the thing that people are supposed to check, if that makes sense. I was thinking it might make it easier when scanning the list for things to jump out at you? I guess if you're checking them off, you're reading through them, but I just think it's kind of like "these are statements that should be true about this content" type of thing.

Reply:
Oooh alright. I think it's more clear with verbs first, because it shows that the person that marks each box has done the task (active vs passive voice)